### PR TITLE
[TACHYON-684] Clean up doc navigation

### PR DESCRIPTION
**Pull request by aurnik**
Added group properties to the doc pages and looped through the groups to generate the navigation instead of hard coding them in for https://tachyon.atlassian.net/browse/TACHYON-684

### CODE REVIEW

**Comment by gpang** [comment]
ok to test

File: docs/Running-Hadoop-MapReduce-on-Tachyon.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ---
 layout: global
 title: Running Hadoop MapReduce on Tachyon
+nickname: Hadoop MapReduce
```
Comment (gpang) [inline docs/Running-Hadoop-MapReduce-on-Tachyon.md]: Is this the title that shows up in the navbar? If so, then Apache projects should be prefixed with "Apache".

File: docs/Running-Spark-on-Tachyon.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 ---
 layout: global
 title: Running Spark on Tachyon
+nickname: Spark
```
Comment (gpang) [inline docs/Running-Spark-on-Tachyon.md]: Is this the title that shows up in the navbar? If so, then Apache projects should be prefixed with "Apache".

File: docs/Setup-UFS.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
 ---
 layout: global
 title: Configure Underlayer Storage System
+nickname: Configure UnderFileSystem
```
Comment (gpang) [inline docs/Setup-UFS.md]: For the nickname and the title, can you use "Under Storage System"?

File: docs/Syncing-the-Underlying-Filesystem.md [hunk 4]
```diff
@@ -1,6 +1,8 @@
 ---
 layout: global
 title: Syncing the Underlayer Filesystem
+nickname: Sync Underlayer FS
```
Comment (gpang) [inline docs/Syncing-the-Underlying-Filesystem.md]: For the nickname and the title, can you use "Under Storage System"?

File: docs/_layouts/global.html [hunk 5]
```diff
@@ -14,11 +14,165 @@
...
+          color: white;
+          padding: 5px 0;
+        }
+        #current-features + ul li p code {
+          background-color: #F7F4F3;
+          border: 1px solid #C4CBCA;
+          font-size: 100%;
+          padding: 0 2px;
+        }
     </style>
```
Comment (gpang) [inline docs/_layouts/global.html]: This doesn't have to be in this PR, but is there a simple way to pull out all of this style into a separate css file?

File: docs/_layouts/global.html [hunk 6]
```diff
@@ -50,90 +205,180 @@
...
-                        <li><a href="Running-Flink-on-Tachyon.html">Apache Flink</a></li>
-                    </ul>
-                </li>
-                <li class="dropdown">
-                    <a href="#" class="dropdown-toggle" data-toggle="dropdown">Deployment<b
+                  {% endif %}
+
+                  <ul class="dropdown-menu">
+                      {% assign currentGroup = site.pages | where:'group',groupName %}
+                      {% assign currentGroup = currentGroup | sort:'priority' %}
```
Comment (gpang) [inline docs/_layouts/global.html]: Is there a way to re-use this block of code for the other drop down menus?

File: docs/Running-Hadoop-MapReduce-on-Tachyon.md (hunk 1 above)
Comment (aurnik) [inline docs/Running-Hadoop-MapReduce-on-Tachyon.md]: Yes it is, I've added the prefix.

File: docs/Running-Spark-on-Tachyon.md (hunk 2 above)
Comment (aurnik) [inline docs/Running-Spark-on-Tachyon.md]: Yes it is, I've added the prefix.

File: docs/Setup-UFS.md (hunk 3 above)
Comment (aurnik) [inline docs/Setup-UFS.md]: Done

File: docs/Syncing-the-Underlying-Filesystem.md (hunk 4 above)
Comment (aurnik) [inline docs/Syncing-the-Underlying-Filesystem.md]: Done

File: docs/_layouts/global.html (hunk 5 above)
Comment (aurnik) [inline docs/_layouts/global.html]: Yes I'll add that into the next PR

File: docs/_layouts/global.html (hunk 6 above)
Comment (aurnik) [inline docs/_layouts/global.html]: I condensed all of the drop down menus into one loop in the next PR

**Comment by gpang** [comment]
LGTM

**Pull request merged**